OktaAuthNative.xcodeproj/project.pbxproj

```diff
@@ -27,7 +27,6 @@
 		F5A9BAD721F9C255002DB437 /* Types.swift in Sources */ = {isa = PBXBuildFile; fileRef = F5A9BAD621F9C255002DB437 /* Types.swift */; };
 		F5CC2AD921D3C27700649C33 /* OktaAPIRequestTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = F5CC2AD821D3C27700649C33 /* OktaAPIRequestTests.swift */; };
 		F5CC2ADB21D5056D00649C33 /* OktaAPITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = F5CC2ADA21D5056D00649C33 /* OktaAPITests.swift */; };
-		FA3E4B0B222003B1004B7468 /* OktaAuthiOSIntegrationTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2FF5C86621D3BA1900EECA75 /* OktaAuthiOSIntegrationTests.swift */; };
```

Comment:
Ildar, why don't we add config on Travis for integration tests?

Reply:
We have to fix master asap to not block merge requests. Regarding integration tests they require environment setup(test instance setup and setting environment variables) which will take some time. I will create separate task for you to manage environment configuration